config/env_blank.json

```diff
@@ -38,5 +38,12 @@
     "user": "",
     "password": ""
   },
-  "APPEND_TABLE_NAMES": ["job_run", "data_source_status"]
+  "APPEND_TABLE_NAMES": [
+    "job_run", "data_source_status",
+    "mivideo_media_started_hourly", "mivideo_media_creation"
```

Comment:
Should probably only include the tables you're adding for the time being.

Reply:
✅Fixed.